doc/services/llext/config.rst

```diff
@@ -0,0 +1,129 @@
+Configuration
+#############
+
+The following Kconfig options are available for the LLEXT subsystem:
+
+.. _llext_kconf_heap:
+
+Heap size
+----------
+
+The LLEXT subsystem needs a static heap to be allocated for extension related
+data. The following option controls this allocation.
+
+:kconfig:option:`CONFIG_LLEXT_HEAP_SIZE`
+
+        Size of the LLEXT heap in kilobytes.
+
+.. note::
+
+   When :ref:`user mode <usermode_api>` is enabled, the heap size must be
+   large enough to allow the extension sections to be allocated with the
+   alignment required by the architecture.
+
+.. _llext_kconf_type:
+
+ELF object type
+---------------
+
+The LLEXT subsystem supports loading different types of extensions; the type
+can be set by choosing among the following Kconfig options:
+
+:kconfig:option:`CONFIG_LLEXT_TYPE_ELF_OBJECT`
+
+        Build and expect relocatable files as binary object type for the LLEXT
+        subsystem. A single compiler invocation is used to generate the object
+        file.
+
+:kconfig:option:`CONFIG_LLEXT_TYPE_ELF_RELOCATABLE`
+
+        Build and expect relocatable (partially linked) files as the binary
+        object type for the LLEXT subsystem. These object files are generated
+        by the linker by combining multiple object files into a single one.
+
+:kconfig:option:`CONFIG_LLEXT_TYPE_ELF_SHAREDLIB`
+
+        Build and expect shared libraries as binary object type for the LLEXT
+        subsystem. The standard linking process is used to generate the shared
+        library from multiple object files.
+
+        This is not currently supported on ARM architectures.
+
+.. _llext_kconf_storage:
```

Comment:
add the two letters to full spell kconfig for the labels here would be just fine by me